### PR TITLE
Auto color correct wrapper

### PR DESCRIPTION
**Pull request by HaleySchuhl**
**Describe your changes**
Adds a wrapper that enables someone to do color corrections in one step. 

**Type of update**
Is this a:
* New feature or feature enhancement

**Associated issues**
Reference associated issue numbers. Does this pull request close any issues?

**Additional context**

**For the reviewer**
See [this page](https://plantcv.readthedocs.io/en/latest/pr_review_process/) for instructions on how to review the pull request.
- [x] PR functionality reviewed in a Jupyter Notebook
- [x] All tests pass
- [x] Test coverage remains 100%
- [x] Documentation tested
- [x] New documentation pages added to `plantcv/mkdocs.yml`
- [x] Changes to function input/output signatures added to `updating.md`
- [x] Code reviewed
- [x] PR approved

### CODE REVIEW

File: docs/updating.md [hunk 1]
```diff
@@ -1210,6 +1210,11 @@ pages for more details on the input and output variable types.
 * pre v3.0dev1: NA
 * post v3.0dev2: determinant, transformation_matrix = **plantcv.transform.calc_transformation_matrix**(*matrix_m, matrix_b*)
 
+#### plantcv.transform.auto_correct_color
+
+* pre v4.6: NA
+* post v4.6: corrected_img = **plantcv.transform.r**(*rgb_img, label=None, **kwargs*)
```
Comment (nfahlgren) [inline docs/updating.md:1216]: Typo in function name

File: tests/plantcv/transform/test_auto_correct_color.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+"""Tests for auto_correct_color."""
+import cv2
+import numpy as np
+from plantcv.plantcv.transform import auto_correct_color
+
```
Comment (nfahlgren) [inline tests/plantcv/transform/test_auto_correct_color.py:5]:
```suggestion


```

**Review by nfahlgren — CHANGES_REQUESTED** [review]

File: docs/transform_auto_correct_color.md
Comment (k034b363) [inline docs/transform_auto_correct_color.md:1]: Looks like the example images for before and after color correction are different samples.

Comment (HaleySchuhl) [reply docs/transform_auto_correct_color.md:1]: Thanks for catching this. I just reused the readable debug image from an existing doc page, and removed the one I had added. Hopefully should be good to go now.

**Review by k034b363 — APPROVED** [review]
Looks good to me! Test notebook with two additional images added to drive.

**Review by nfahlgren — APPROVED** [review]

**Pull request merged**